google_monitoring/uptime_checks.tf

```diff
@@ -7,10 +7,23 @@ resource "google_monitoring_uptime_check_config" "https" {
   user_labels  = each.value.user_labels
 
   http_check {
-    path         = each.value.path
-    port         = 443
-    use_ssl      = true
-    validate_ssl = true
+    path                = each.value.path
+    port                = 443
+    request_method      = each.value.request_method
+    use_ssl             = true
+    validate_ssl        = true
+    content_type        = lookup(each.value, "content_type", null)
+    custom_content_type = lookup(each.value, "custom_content_type", null)
+    body                = lookup(each.value, "body", null)
+  }
+
+  dynamic "content_matchers" {
+    for_each = each.value.content_matchers
+
+    content {
+      content = content_matchers.value.content
+      matcher = content_matchers.value.matcher
+    }
```

Comment:
Same situation here -- except the [`content_matchers` docs](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/monitoring_uptime_check_config#content_matchers) explicitly state that:

> This field is optional and should only be specified if a content match is required.

Reply:
Now that I've had more time to try this out and inspect the code, I see you default `content_matchers` to an empty list, `[]`. So I imagine that means we never pass this argument in either as a result. The fact that my uptime check PR using this config is still working after `atlantis apply` supports that.